.github/scripts/lintrunner.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -x
+
+# The generic Linux job chooses to use base env, not the one setup by the image
+CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+conda activate "${CONDA_ENV}"
+
+CACHE_DIRECTORY="/tmp/.lintbin"
```

Comment:
Is this directory created by docker build? In that case, woudln't it be missing if other docker image is used for non-clang lints?

Reply:
I see .lintbin mentioned in the Docker file for both the cuda and the non cuda builds